docs/approvals/jira-service-management/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Jira Service Management Integration
+description: Octopus Deploy can integrate with your Jira Service Management instance for deployment 
+control using Change Requests/Issues
+position: 10
+---
+
+:::warning
+The Jira Service Management (JSM) Integration feature is only available as an EAP for version 
+**2022.3.
+12101** or later and it requires a specific feature license to use it. To request a 
+license register for the [JSM Early Access Program](https://octopusdeploy.
+typeform.com/servicenow-eap)
+:::
+
+## Overview
+
+The Octopus Deploy/JSM integration allows users to block the execution of 
+specifically configured deployments unless they have a corresponding approved JSM  **Change 
+Request** (aka issue).
+
+To enable this behavior, both the Octopus Project and Environment you are deploying to must be 
+configured and the JSM configuration is set up before deployments can be managed.
+
+| Project | Environment | Outcome |
+|--|--|--|
+| Change controlled| Change controlled| Approval required for deployment |
+| **_Not_** Change controlled| Change controlled| No approval required |
+| Change controlled| **_Not_** Change controlled| No approval required |
+
+## Getting started
+
+The JSM integration requires Octopus **2022.3.12101** or later and an Octopus license with the
+JSM Integration feature enabled.
+
+Before you can use the Octopus Deploy/JSM integration, you'll need to:
+
+1. Create a service account in JSM for use by Octopus
+1. Request and install a new Octopus license required to enable the JSM feature.
+1. Configure a connection from Octopus to JSM.
+1. Configure which deployments require an approved CR.
+
+### Configuring Jira Service Management
+
+:::hint
+The instructions in this section will require a JSM Administrator.
+:::
+
+The Octopus Deploy / JSM integration requires security configuration in your target JSM instance.
```

Comment:
nit: on line 18 and 36 we don't put spaces around the `/` but we do here (consistency)

Reply:
done